### PR TITLE
Adding Sanjose to hometown

### PR DESCRIPTION
**Pull request by nughma**
This is a pull request

### CODE REVIEW

**Comment by githubteacher** [comment]
@nughma Looks good! Care to add any info about what to do in San Jose?

File: nughma-sanjose.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a file created for some random contents in order to learn collaboration on github.
```
Comment (CSTaylorJr) [inline nughma-sanjose.md:1]: This is a random comment for the random file. :smile:

**Comment by githubteacher** [comment]
👍 Looks good! :shipit:

**Comment by tharappel** [comment]
:+1:  looks good

File: nughma-sanjose.md (hunk 1 above)
Comment (CSTaylorJr) [reply nughma-sanjose.md:1]: :+1:

**Pull request merged**